include/walker-dropdown.php

```diff
@@ -35,7 +35,7 @@ public function start_el( &$output, $element, $depth = 0, $args = array(), $curr
 		$output .= sprintf(
 			"\t" . '<option value="%1$s"%2$s%3$s>%4$s</option>' . "\n",
 			'url' === $value_type ? esc_url( $element->$value_type ) : esc_attr( $element->$value_type ),
-			method_exists( $element, 'get_locale' ) ? sprintf( ' lang="%s"', esc_attr( $element->get_locale( 'display' ) ) ) : '',
+			! empty( $element->locale ) ? sprintf( ' lang="%s"', esc_attr( str_replace( '_', '-', $element->locale ) ) ) : '',
```

Comment:
It seems to me that `$element->locale` is already what we are expectng: https://github.com/polylang/polylang/blob/3.2.2/include/switcher.php#L128

Reply:
> It seems to me that `$element->locale` is already what we are expectng: https://github.com/polylang/polylang/blob/3.2.2/include/switcher.php#L128

Yes you're right and also already converted if required: https://github.com/polylang/polylang/blob/3.2.2/include/language.php#L217